### PR TITLE
shim if vttcue isn't available

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Review by brandonocasey — APPROVED** [review]

**Comment by brandonocasey** [comment]
seems better then my implementation in #9 , I have one comment LGTM otherwise

File: lib/browser-index.js [hunk 1]
```diff
@@ -33,11 +33,15 @@ var oldVTTCue = window.VTTCue;
...
-  vttjs.VTTCue = cueShim;
-  vttjs.VTTRegion = regionShim;
+  window.VTTCue = cueShim;
+  window.VTTRegion = regionShim;
 };
 
 vttjs.restore = function() {
-  vttjs.VTTCue = oldVTTCue;
-  vttjs.VTTRegion = oldVTTRegion;
+  window.VTTCue = oldVTTCue;
```
Comment (brandonocasey) [inline lib/browser-index.js]: maybe it would be better to rename this to, nativeVTTRegion & nativeVTTCue?

Comment (gkatsev) [reply lib/browser-index.js]: makes sense.

Comment (gkatsev) [reply lib/browser-index.js]: Updated.

**Review by brandonocasey — APPROVED** [review]

**Pull request merged**